nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -167,3 +169,17 @@ def mean_minmax_statistic_collector(nncf_graph: NNCFGraph,
     @staticmethod
     def get_weight_tensor_port_id(node: NNCFNode) -> int:
         return node.metatype.weight_definitions.weight_port_id
+
+    @staticmethod
+    def get_model_type_ignore_scope(model_type: ModelType) -> IgnoredScope:
+        if model_type == ModelType.TRANSFORMER:
+            return IgnoredScope(types=["Add", "Pow", "Squeeze", "Mul", "Div", "Sub", "ReduceMean"])
```

Comment:
I suggest to use metatypes here as in the torch backend for consistency.

Reply:
Done